### PR TITLE
Fixes #433 : Built activities.html

### LINKED ISSUE #433: Adding our website Issues Page using Github API
### Expected results

Adding our website Issues Page using Github API, as in our previous **GCI** Websites.

### Actual results

No such page exists for now.

## Would you like to work on this

@0x48piraj and @duttaditya18 are working on this. :heart:

### PR DESCRIPTION
**Pull request by 0x48piraj**
# Fixes #433
- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- I came with an idea to make a `issues.html` which will contain all the issues and PRs of our website, as previous GCI websites. :heart: 
- Built **activities.html,** added a button, and Implemented Github API to list `repo` PRs and issues. :sparkles: 
- @duttaditya18 re-worked on the UI Part, Helped me to list the issues correctly, Added Animations, ie. beautified the `activities.html` :100: 

# Screenshot : 

image

_**Preview - https://0x48piraj.github.io/gci17.fossasia.org/activities.html**_

### CODE REVIEW

**Comment by ba11b0y** [comment]
Great work!
But from now onwards, before working on adding new features try gathering responses and reviews from everyone on the issue, so that if in any case the feature is not much appreciated, it can be dropped then and there.
Please wait while everyone checks this.

**Comment by nzec** [comment]
@invinciblycool We will continue to work on this. We have color coded issues and pull requests panned!

**Comment by ba11b0y** [comment]
Great, but add a WIP label if you plan to work on this PR. 
Good going

**Comment by yashLadha** [comment]
Great work @0x48piraj . It would be better if we can have pagination of the responses, there can be a button for limiting the content on the main page.

**Comment by 0x48piraj** [comment]
Got it @yashLadha :100: 

Will implement **Paginated Responses**, using some creative animations. :sparkles:

**Comment by abishekvashok** [comment]
Also, pls centerly align the content.

**Comment by 0x48piraj** [comment]
@abishekvashok 
After it's merged, we will add a whole cool UI, with **Paginated Responses** as @yashLadha suggested. :+1:

**Comment by yashLadha** [comment]
Great! Appreciate your enthusiasm.

**Comment by nzec** [comment]
@yashLadha You said that we needed to do Paginated Responses right? So like there are different pages. I think we can use jekyll for this.

**Comment by yashLadha** [comment]
It would be better if the paginated responses are on the same and we can
use JavaScript to show the responses on the same page without refresh.

**Comment by abishekvashok** [comment]
I think it's overflowing on mobile. Rest great work :) 👍
screenshot_2017-12-23-19-33-30-979_com android chrome

**Comment by 0x48piraj** [comment]
It's just a start @abishekvashok, all pages are currently not looking good in mobile browsers.
You can see, most (if not all) sub-links are broken just like this.
We'll have to optimize, but first we should at least get it working in web-browsers :+1:

**Comment by abishekvashok** [comment]
Keep going :)👍👍👍

**Comment by supunTE** [comment]
merge this with upstream

**Review by supunTE — APPROVED** [review]
Great work! squash it. Can you add current PR showing page, that will be excellent! LGTA!!!

**Comment by supunTE** [comment]
erragain
this render problem that you said; showing `---`. again there pls see it

**Comment by 0x48piraj** [comment]
@STEdirisuriya See the PR before making a comment/review.
The review you did, was of another PR. :+1: 

> Ref : https://github.com/fossasia/gci17.fossasia.org/pull/438

**Review by akshatnitd — COMMENTED** [review]
@0x48piraj Please squash your last four commits :)

**Review by akshatnitd — APPROVED** [review]
This PR looks good to me. Further iterations can target improvements on adding Pagination feature and making it responsive.

Btw, @0x48piraj @duttaditya18 Great work on the PR :+1:

**Comment by 0x48piraj** [comment]
Ping @fossasia/website-admins :100: 

Ping (as suggested by another mentor)
@ParthS007 
@vaibhavsingh97 
@isuruAb

**Review by ParthS007 — COMMENTED** [review]
Good work @0x48piraj @duttaditya18  ! Keep it up . Please squash your commits and your PR is ready to merge and create sub-issues which are related to it 👍

File: js/contributors.js [hunk 1]
```diff
@@ -49,4 +49,23 @@ $(document).ready(function(){
       $("#repos").append(html);
     });
   });
+
+  $.getJSON("https://api.github.com/repos/fossasia/gci17.fossasia.org/issues?state=open", function (allIssues) {
+    if(allIssues==""||!allIssues||allIssues==0||allIssues=="undefined"){
+      $(".issue-container").append("<li class='mi'>Sorry there are no issues labeled feature currently opened at gci15.fossasia.org</li>");
```
Comment (sayak119) [inline js/contributors.js]: gci17

**Review by sayak119 — APPROVED** [review]
Good work LGTM

File: js/contributors.js (hunk 1 above)
Comment (0x48piraj) [reply js/contributors.js]: On it :+1:

**Comment by akshatnitd** [comment]
Merging :rocket: Please squash the commits from next time. Merging this time, as you are facing some problems this time. Also, make it a habit of amending the commits when any change is requested, so that an additional commit isn't created :smile:

**Comment by 0x48piraj** [comment]
Aye Aye Captain ! :100:

**Comment by vaibhavsingh97** [comment]
@0x48piraj Good Work !! 🥇

**Comment by 0x48piraj** [comment]
Thanks to all of you, my fellow buddies and our awesome mentors! :sparkles:

**Pull request merged**